### PR TITLE
Ex. 8 Join: People

### PR DESCRIPTION
**Pull request by KinkyHairCoder**

### CODE REVIEW

File: Assignments/stu1/queries.sql [hunk 1]
```diff
@@ -29,7 +29,11 @@ select count(*) from person where person_type_id = (2);
 /**
  * Ex. 8
  */
-
+select 
+  first_name, last_name, person_type
+from 
+  person 
+inner join person_type on person.person_id = person_type.person_type_id;
```
Comment (mburolla) [inline Assignments/stu1/queries.sql]: Hum... `person.person_id = person_type.person_type_id`

This looks a bit off.  :-)

Comment (KinkyHairCoder) [reply Assignments/stu1/queries.sql]: It works but I changed it. Also, the last push will probably look messy.

Comment (mburolla) [reply Assignments/stu1/queries.sql]: Hum, I'm not seeing your changes in GitHub.

Comment (KinkyHairCoder) [reply Assignments/stu1/queries.sql]: It should show now.

File: Assignments/stu1/queries.sql [hunk 2]
```diff
@@ -30,11 +30,10 @@ select count(*) from person where person_type_id = (2);
...
  */
 select 
-  first_name, last_name, person_type
+  p.first_name, p.last_name, per.person_type
 from 
-  person 
-inner join person_type on person.person_id = person_type.person_type_id;
-
+  person p
+inner join person_type per on p.person_id = per.person_type_id;
```
Comment (mburolla) [inline Assignments/stu1/queries.sql]: Well... do we really want to join on `person_id` and `person_type_id`?  (The answer is no).  :-)

File: Assignments/stu1/queries.sql [hunk 3]
```diff
@@ -33,7 +33,7 @@ select
   p.first_name, p.last_name, per.person_type
 from 
   person p
-inner join person_type per on p.person_id = per.person_type_id;
+inner join person_type per on p.book_store_id = per.person_type_id;
```
Comment (mburolla) [inline Assignments/stu1/queries.sql]: Nope.

**Pull request merged**